pages/vi/profiles/joelschwartz.md

```diff
@@ -0,0 +1,13 @@
+
+1. My name is Joel Schwartz
+2. I am from Mansfield Ohio 
+3. my time zone is eastern standard time
+
+![adirondack mountains view](https://www.wildernessphotographs.com/images/xl/Boreas-Mt-Treetop-crop-of-Elk-Lake-High-Peaks-_1000px.jpg)
+
+* The operating system that I use is windows ten home 1803.
+* I am familiar with *HTML5*, *CSS3*,  **Javascript**, **SQL**, **C#**, *ASP.NET SPA framework*, and *PHP*.
```

Comment:
You seem to have tried using multiple markdown. Which is good but having different markdown on same context does not look nice. Using markdown is part of this step but so is having good profile page of yours.
So can you make consistent markdown on list of language sets.